cpp/oneapi/dal/algo/svm/backend/cpu/train_kernel.cpp

```diff
@@ -53,12 +53,45 @@ static train_result call_daal_kernel(const context_cpu& ctx,
 
     // TODO: data is table, not a homogen_table. Think better about accessor - is it enough to have just a row_accessor?
     auto arr_data    = row_accessor<const Float>{ data }.pull();
-    auto arr_label   = row_accessor<const Float>{ labels }.pull();
     auto arr_weights = row_accessor<const Float>{ weights }.pull();
 
+    auto arr_label     = row_accessor<const Float>{ labels }.pull();
+    auto arr_new_label = array<Float>::empty(row_count * 1);
+
+    arr_label.need_mutable_data();
+    Float value_first_class_label  = arr_label[0];
+    Float value_second_class_label = arr_label[1];
+
+    arr_new_label[0] = Float(-1.0);
```

Comment:
I think you should create constants somewhere in svm classes rather than use magic constants.